plugins/experimental/parent_select/parent_select.cc

```diff
@@ -373,14 +373,14 @@ TSRemapNewInstance(int argc, char *argv[], void **ih, char *errbuff, int errbuff
 
   std::unique_ptr<TSNextHopSelectionStrategy> strategy;
 
-  for (auto it = file_strategies.begin(); it != file_strategies.end(); it++) {
+  for (auto &file_strategie : file_strategies) {
```

Comment:
Ha: strategie. :) It took the 's' off to make it singular.

Beyond that, it looks like first and second are both used in the loop. Since we're using range-based for, can we used a structured bind on this map? Something like:

`for (auto &[name, strategy]: file_strategies)`